src/fontra/views/editor/scene-model.js

```diff
@@ -706,6 +719,47 @@ export class SceneModel {
   //fontGuidelineSelectionAtPoint(point, size) {
   // }
 
+  imageSelectionAtPoint(point) {
+    if (
+      !this.editorController.visualizationLayersSettings.model[
+        "fontra.background.image"
+      ]
+    ) {
+      return new Set();
+    }
+
+    const positionedGlyph = this.getSelectedPositionedGlyph();
+    if (!positionedGlyph) {
+      return new Set();
+    }
+
+    const image = positionedGlyph.glyph.image;
+    if (!image) {
+      return new Set();
+    }
+
+    const x = point.x - positionedGlyph.x;
+    const y = point.y - positionedGlyph.y;
+
+    const img = new Image();
+    img.src = "data:image/jpg;base64," + this.fontController.getImage(image.fileName);
+
+    const sx = image.transformation.translateX;
```

Comment:
The way to do this is to convert the decomposed transform to a "normal" transform with `decomposedToTransform(image.transformation)`, and then you use the `t.transformPoint()` method for the four corners.